benchmarks/python/test_dropout_layernorm_bwd.py

```diff
@@ -195,6 +196,8 @@ def test_dropout_layernorm_bwd_nvf_benchmark(
 @pytest.mark.parametrize("executor", DEFAULT_EXECUTORS)
 @pytest.mark.parametrize("size", generate_input_sizes(dims=2))
 @pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+@pytest.mark.inner_outer_persistent
+@pytest.mark.inner_persistent
 def test_dropout_layernorm_bwd_baseline_benchmark(
```

Comment:
For baseline benchmarks, it only makes sense to add a scheduler marker when the executor is nvFuser. nvFuser is only used when the executor is `thunder`